cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -247,6 +397,15 @@ else()
   set(BROTLI_VENDORED 0)
 endif()
 
+if (MSVC)
+  ExternalProject_Get_Property(brotli_ep SOURCE_DIR)
+
+  ExternalProject_Add_Step(brotli_ep headers_copy
+    COMMAND xcopy /E /I include ..\\..\\..\\brotli_ep\\src\\brotli_ep-install\\include /Y
+    DEPENDEES build
+    WORKING_DIRECTORY ${SOURCE_DIR})
```

Comment:
Install step available, but it has issues on Windows. Also their docs mention cmake build instructions only for unix. Have reported https://github.com/google/brotli/issues/543